src/lifecycles/bootstrap.js

```diff
@@ -6,21 +6,16 @@ import {
 } from "../applications/app.helpers.js";
 import { reasonableTime } from "../applications/timeouts.js";
 import { handleAppError, transformErr } from "../applications/app-errors.js";
-import { getProps } from "./prop.helpers.js";
 
-export function toBootstrapPromise(appOrParcel, hardFail = false) {
```

Comment:
Using the default argument here added a fair amount of bytes and was unnecessary